client/src/containers/dashboard/dashboard.js

```diff
@@ -17,7 +17,7 @@ export default class Dashboard extends React.Component {
           <SubmissionInfo />
         </div>
         <div className="group-info-grid">
-          <Profile />
+          <Profile teamName={"testTeam"} primaryContactName={"testName"} primaryContactEmail={"testEmail"} />
```

Comment:
To make this a little bit easier to transition to non test data, go ahead and create some variables for your test values. Something like:
```
const teamName = "testTeam"
```
Go ahead and do that for all 3 of the props you're pasting in. This way, we can easily transfer to using real data.